apps/web/pages/[user].tsx

```diff
@@ -1,3 +1,4 @@
+import * as Sentry from "@sentry/node";
```

Comment:
Ideally we don't import Sentry all across the app but instead we have a generic logger/exception handler that we can shield this dependency behind which would allow us to quickly swap out Sentry in the future if desired. Let's quickly add this, please. 🙏